openpype/settings/defaults/project_settings/nuke.json

```diff
@@ -8,6 +8,13 @@
             "build_workfile": "ctrl+alt+b"
         }
     },
+    "nuke-dirmap": {
+        "enabled": true,
+        "paths": {
+            "source-path": [],
```

Comment:
is any input processed by environment var solving or even solved by settings key as it is used in anatomy? So we could use `{root}` in source paths?

Reply:
Currently no, but I would put that into new PR as it would be necessary to implement it also for Maya.